app/src/main/java/net/bible/service/sword/SwordContentFacade.java

```diff
@@ -231,8 +231,8 @@ public String getCanonicalText(Book book, Key key) throws NoSuchKeyException, Bo
     	try {
 			BookData data = new BookData(book, key);
 			SAXEventProvider osissep = data.getSAXEventProvider();
-		
-			ContentHandler osisHandler = new OsisToCanonicalTextSaxHandler();
+
+			ContentHandler osisHandler = new OsisToCopyTextSaxHandler(key.getOsisID().trim().contains(" "));
```

Comment:
Hmm, can a single key ever contain multiple verses and space?  

Could this Key be also VerseRange? Then there could be also verse ranges like Gen1.1-Gen.1.2

Reply:
From my testing when multiple verses are selected, the key value becomes "Gen.1.1 Gen.1.2 Gen.1.3" etc. I couldn't find another easy way yet to get verse count from key but maybe you know how. I actually find that the calling function shareVerse (PageControl.java line 97) passes a verseRange object to the key.
I could use verseRange and use verseRange.toVerseArray().length, but don't know what is the best way.